networks/osmosis-1/upgrades/v28/mainnet/v28_binaries.json

```diff
@@ -0,0 +1,6 @@
+{
+  "binaries": {
+    "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v28.0.0/osmosisd-28.0.0-linux-amd64?checksum=TODO",
+    "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v28.0.0/osmosisd-28.0.0-linux-arm64?checksum=TODO"
```

Comment:
Let's do the release first and get the proper checksum in place

Reply:
:heavy_plus_sign: